docs/community/Issue-Triaging.md

```diff
@@ -0,0 +1,108 @@
+|[`feature-request`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | request for a new feature|
+|[`tech-debt`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | improve the implementation/architecture|
+|[`engineering`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | issues related to our engineering system or our processes|
+|[`polish`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | a feature could be improved, but not necessarily a bug|
+|[`*question`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | we should direct questions to StackOverflow|
+
+### Feature Areas Labels
+
+Each issue must have a **feature area** label. 
+
+|Feature Area|Description|
+|---|---|
+|[`rendering`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Rendering |
+|[`shader`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Shader |
+|[`styling`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Styling |
+
+### Platform Labels
+
+Each issue must have a **platform** label. 
+
+|Platform|Description|
+|---|---|
+|[`ios-simulator`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | iOS Simulator|
+|[`ios-device`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Physical iOS device|
+|[`android-simulator`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Android Simulator |
+|[`android-device`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Physical Android device|
+|[`macos-x86`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Mac OS |
+|[`macos-arm`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | M-series chips Mac OS |
+|[`linux`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Linux |
+|[`qt`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Qt |
+|[`node`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Node.JS |
+|[`all-platforms`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Across all platforms |
+
+### Important Issues
+
+The TSC reviewers may add `important` labels to issues that
+- Critical security vulnerabilities
+- Stability or performance issue that makes a feature unusable
+
+### Other labels
+
+If issues are suitable for beginners we may add the `good-first-issue` label and we add code pointers that help beginners to get started with a PR.
+
+Sometime, we get issues that we can't or don't have the time to reproduce due to the complexity or time requirements of the setup but that we indeed suspect to be issues. We label those issues with `investigation-wanted`. What we are looking for is help in reproducing and analyzing the issue.
+
+
+## Community Review
+
+### Accepting issue
+Community members can "up-vote" an issue (adding a GitHub +1/"👍" reaction to the issue description). If an issue surpasses the 20 up-votes in 60 days, the issue will be accepted.
```

Comment:
i don't think a formal "20 up-votes in 60" is a good measure. Each repo/target may have very different number of interest parties, plus there could be a lot of "yei" votes without explanation and a single but very clearly explained "no" vote that should overpower it.  We had similar issues with Wikipedia, and usually direct voting like this don't work well. Instead, the community simply tries to get to a consensus by discussing it, and usually arrives to a solution agreed by everyone. If not, there is always steering committee or the board to decide highly conflicted topics (hopefully very few)